test/type_traits.cpp

```diff
@@ -355,74 +355,242 @@ TEST_CASE("is_unsigned", "[meta.unary.prop]") {
 TEST_CASE("is_constructible", "[meta.unary.prop]") {
   CHECK(
       std::is_base_of<slb::true_type, slb::is_constructible<int, int>>::value);
+
+  /* destructible */ {
+    struct ND {
+      ND(int) {}
+      ~ND() = delete;
+    };
+
+    CHECK(std::is_base_of<slb::false_type,
+                          slb::is_constructible<ND, int>>::value);
+  }
 }
 
 // template<class T> struct is_default_constructible;
 TEST_CASE("is_default_constructible", "[meta.unary.prop]") {
   CHECK(std::is_base_of<slb::true_type,
                         slb::is_default_constructible<int>>::value);
+
+  /* destructible */ {
+    struct ND {
+      ND() {};
```

Comment:
The extraneous semi-colon is still here.

Reply:
Oh, so it wasn't (just) about spacing...

Reply:
'twas both 😄